packages/contracts/contracts/maci/Poll.sol

```diff
@@ -28,6 +37,20 @@ contract Poll is Ownable, BasePoll {
     BasePoll(duration, treeDepths, coordinatorPubKey, extContracts, emptyBallotRoot)
   {}
 
+  /// @notice Set poll registry.
+  /// @param registryAddress The registry address
+  function setRegistry(address registryAddress) public onlyOwner {
```

Comment:
It's not possible to override init because it doesn't have any params. It will be possible if override modifier will be removed.

Reply:
> It will be possible if override modifier will be removed.

so if it's not virtual you can make another with the same name? but then there will be 2 functions technically?

Reply:
Yes, there will be 2 functions but I think it's the same as we have `setRegistry` and `init`. Both methods are called in `MACI.sol` when poll is initialized.

Reply:
yea good then makes sense to keep as you did then